content/en/observability_pipelines/legacy/production_deployment_overview.md

```diff
@@ -89,29 +88,6 @@ Data processing is shifted off your nodes and onto remote aggregator nodes. Remo
 
 See [Aggregator Architecture][5] for more details.
 
-## Buffering data
```

Comment:
I think removing this for now is good, but we should write up something about the "built in default" buffering (and remove language about being able to configure it). 

We can probably pull something together from here: https://vector.dev/docs/about/under-the-hood/architecture/buffering-model/ then likely have eng review it. 

I can draft something in a google doc if you like

Reply:
Sure, if you can draft something that'd be great. Thanks!

Reply:
Check it out here May: https://docs.google.com/document/d/1UKspJYeXbvY5tFdlJUJgjvJ2ngLfg36MAkKVz6YCQvo/edit

Want to post in the #obs-pipelines-backroom channel for a look?

Reply:
Oh jeez! These are the "legacy" 1.0 docs. We can disregard. Leave this as-is.

Reply:
Sorry that was my fault, I thought I had stuck to the current docs. I reverted the change.